components/Pages/CarePackages/ReviewPackageDetails/index.js

```diff
@@ -14,85 +14,106 @@ const links = [
+            }) => {
+              return (
                 <Container key={itemId} className="review-package-details__cost-info-item">
-                  <ReviewPackageInfo containerId={itemId} headerTitle={headerTitle} items={items} />
-                  {costOfPlacement && (
-                    <p className="brokerage__cost-of-placement">
-                      Cost of placement
-                      <span className="text-lbh-f01 font-weight-bold">
-                        {currency.euro}
-                        {costOfPlacement}
-                      </span>
-                    </p>
-                  )}
-                  {totalCost && <BrokerageBorderCost totalCost={totalCost} totalCostHeader={totalCostHeader} />}
-                  {totalCostComponent && totalCostComponent}
-                  {totalCost && (
-                    <Container className="review-package-details__items-actions" display="flex">
-                      <p onClick={redirectToBrokerPackage} className="link-button">
-                        Edit
-                      </p>
-                      <p onClick={redirectToBrokerPackage} className="link-button red">
-                        Remove
-                      </p>
-                    </Container>
-                  )}
+                  <PackageInfo details={details} containerId={itemId} headerTitle={headerTitle} items={items}/>
+                  {!!costOfPlacement &&
+                  <p className="brokerage__cost-of-placement">
+                    Cost of placement
+                    <span className="text-lbh-f01 font-weight-bold">{currency.euro}{costOfPlacement.toFixed(2)}</span>
+                  </p>
+                  }
+                  {!!totalCost && <BrokerageBorderCost totalCost={totalCost} totalCostHeader={totalCostHeader}/>}
+                  {totalCostComponent}
+                  {!!totalCost &&
+                  <Container className="review-package-details__items-actions" display="flex">
+                    <p onClick={() => redirectToBrokerPackage()} className="link-button">Edit</p>
+                    <p onClick={() => redirectToBrokerPackage()} className="link-button red">Remove</p>
+                  </Container>
+                  }
                 </Container>
-              )
-            )}
+              );
+            })}
             <Container className="review-package-details__summary">
-              <h3 id="summary" className="font-weight-bold">
-                Summary
-              </h3>
+              <h3 id="summary" className="font-weight-bold">Summary</h3>
               {summary.map(({ key, value, className, id }) => (
                 <BrokerageTotalCost key={id} value={value} name={key} className={className} />
               ))}
             </Container>
             <Container className="review-package-details__actions" display="flex">
-              <Button handler={router.back}>Back</Button>
+              <Button handler={goBack}>Back</Button>
```

Comment:
You can pass `router.back` directly without creating a new function
```suggestion
              <Button handler={router.back}>Back</Button>
```